internal/controller/openslo/slo_controller.go

```diff
@@ -119,16 +119,17 @@ func (r *SLOReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.R
 		log.Info("PrometheusRule not found. Let's make one.")
 		prometheusRule, err = helpers.CreatePrometheusRule(slo, sli)
 		if err != nil {
-			err = utils.UpdateStatus(ctx, slo, r.Client, "Ready", metav1.ConditionFalse, "Failed to create Prometheus Rule")
-			if err != nil {
-				log.Error(err, "Failed to update SLO status")
-				return ctrl.Result{}, err
+			r.Recorder.Event(slo, "Warning", "FailedToCreatePrometheusRule", "Failed to create Prometheus Rule")
+			errUpdateStatus := utils.UpdateStatus(ctx, slo, r.Client, "Ready", metav1.ConditionFalse, fmt.Sprintf("Failed to create new Prometheus Rule: %v", err))
+			if errUpdateStatus != nil {
+				log.Error(errUpdateStatus, "Failed to update SLO status")
+				return ctrl.Result{}, errUpdateStatus
 			}
-			log.Error(err, "Failed to create new PrometheusRule")
-			return ctrl.Result{}, err
+			log.Info(fmt.Sprintf("Failed to create new Prometheus Rule: %v", err))
```

Comment:
And I am certainly not OK with logging errors at `Info` level.

Reply:
we don't need the actual error though... this is here just because before this, we log that we wanna create PrometheusRule that we don't have, so this is a "result" kind of feedback

IMO, I would just delete both

Reply:
@Hy3n4 so do we wanna move most logging to debug and focus more on events/statuses? 

IMO it would clean up the log quite a lot?

Reply:
when an action results in some unwanted state, Yes. But I would keep the Info level where it makes sense.

Reply:
We don't have a Debug level, so I used `log.V(4).Info()`